### DISCUSSION TITLE
`bury: true` support in sitemap

### DISCUSSION THREAD

**Discussion by amitu** [fastn-stack/fastn#697] [Ideas & RFCs]
In sitemap, sometimes we want some sections, or some sub-section/toc-item to be "buried" by default. If we have many sections/sub-sections/toc-items at any level, it makes sense to bury them. When something is buried, we will show a single (...) icon for every buried item at that level, and when user clicks on (...) the buried items would be shown.

```ftd
-- fpm.sitemap:

# Home
# Docs
## Data
## UI
## API
   bury: true

- foo
- bar
- baz
  bury: true
# Learn
# Contribute
  bury: true
```

In above example we see bury at section, sub-section and toc levels.

**Comment by ADITIRAI290698** [comment]
Owner: @ADITIRAI290698

Owner: @AviralVerma13

Started On: 7th Nov 2022

Reference Discussion : https://github.com/ftd-lang/fpm/discussions/604
Reference PR : https://github.com/ftd-lang/fpm/pull/594

## 7th Novemeber
- Going through the PR : https://github.com/ftd-lang/fpm/pull/594
